src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -80,6 +84,8 @@ abstract protected function status_single( $args );
 
 	abstract protected function install_from_repo( $slug, $assoc_args );
 
+	abstract public function activate( $args, $assoc_args = [] );
```

Comment:
Should probably stick to consistent command signature as well.

Reply:
Done!

Reply:
So `$assoc_args` is sometimes optional because the class is extended. The method is sometimes called with 1 parameter from within the sub class itself.